services/src/pro/contexts/postgres.rs

```diff
@@ -1071,6 +1071,8 @@ mod tests {
                     columns: [("foo".to_owned(), FeatureDataType::Float)]
                         .into_iter()
                         .collect(),
+                    time: None,
+                    bbox: None,
```

Comment:
and here?

Reply:
also same here, because we just store the dataset as a json in the db